aiogram/types/message.py

```diff
@@ -968,6 +968,9 @@ async def answer_location(
             None,
         ] = None,
         reply: base.Boolean = False,
+        horizontal_accuracy: typing.Optional[base.Float] = None,
+        heading: typing.Optional[base.Integer] = None,
+        proximity_alert_radius: typing.Optional[base.Integer] = None,
```

Comment:
I think it'd be better to leave `reply` the last arg as it is not really related to the API but to our shortcuts. Also `allow_sending_without_reply` is API 5.0 change, so IMO we can really put these new args before `reply_markup` as I don't think there is anyone who does smth like `message.answer_location(42.0, 42.0, 60, True, False, keyboard, False)`, otherwise it's weird, isn't it? :smile: 

TL;DR I guess `reply_markup` is usually used as a keyword-arg, so new args would fit best before `reply_markup`